src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/LicenceController.kt

```diff
@@ -1063,4 +1065,42 @@ class LicenceController(
   ) {
     updateSentenceDateService.updateSentenceDates(licenceId, request)
   }
+
+  @Tag(name = Tags.LICENCES)
+  @GetMapping(value = ["/getCutOffDate"])
+  @PreAuthorize("hasAnyRole('SYSTEM_USER', 'CVL_ADMIN')")
+  @ResponseBody
+  @Operation(
+    summary = "Get cut-off date for licence time out",
+    description = "Returns a cut-off date to filter hardstop licences. " +
+      "Requires ROLE_SYSTEM_USER or ROLE_CVL_ADMIN.",
+    security = [SecurityRequirement(name = "ROLE_SYSTEM_USER"), SecurityRequirement(name = "ROLE_CVL_ADMIN")],
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "Licence found",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = Licence::class))],
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorised, requires a valid Oauth2 token",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Forbidden, requires an appropriate role",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "404",
+        description = "The licence for this ID was not found.",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+    ],
+  )
+  fun getCutOffDate(): HardStopCutoffDate {
```

Comment:
`getHardStopCutoffDate`

Reply:
Not using this anymore, directly calling releasedate service from dates controller.